signer/src/request_decider.rs

```diff
@@ -126,17 +126,35 @@ where
             .await?;
 
         for deposit_request in deposit_requests {
-            self.handle_pending_deposit_request(deposit_request, &chain_tip)
-                .await?;
+            let outpoint = deposit_request.outpoint();
+            let _ = self
```

Comment:
Changed this logic so that we don't bail at the first sign of trouble (i.e. we keep processing the remaining requests).

Reply:
sanity check: are we potentially breaking any assumptions in the code by only processing and propagating a subset of the deposit/withdrawal requests in a block? I don't think so, since without this change, if we fail mid-iteration, we'd still end up in a similar state. Curious if you have thoughts or know of any cases where this might cause issues.

Reply:
Dunno why, but it looks like a good idea not to halt everything for a single request error :) I can't think at any broken assuptions either, so we are good!